src/main/java/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,50 @@
+package bookstore.controller;
+
+import bookstore.dto.BookDto;
+import bookstore.dto.CreateBookRequestDto;
+import bookstore.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/books")
+public class BookController {
+    private final BookService bookService;
+
+    @GetMapping
+    public List<BookDto> getAll() {
+        return bookService.findAll();
+    }
+
+    @PostMapping
+    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+        return bookService.save(requestDto);
+    }
+
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.getById(id);
+    }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
If you add @ResponseStatus(HttpStatus.NO_CONTENT) for one endpoint,  you should add it for all other endpoints